src/main/scala/za/co/absa/abris/avro/parsing/utils/AvroSchemaUtils.scala

```diff
@@ -56,45 +57,49 @@ object AvroSchemaUtils {
     try IOUtils.readLines(stream).asScala.mkString("\n") finally stream.close()
   }
 
+  def toAvroSchema(
+                    dataFrame: DataFrame
+                  ): Schema = toAvroSchema(dataFrame, "topLevelRecord", "")
+
+
+  def toAvroSchema(
+                    dataFrame: DataFrame,
+                    columnNames: Seq[String]
+                  ): Schema = toAvroSchema(dataFrame, columnNames, "topLevelRecord", "", DefaultAvroSchemaConverter())
+
+  def toAvroSchema(
+                    dataFrame: DataFrame,
+                    recordName: String,
+                    nameSpace: String
+                  ): Schema =
+    toAvroSchema(dataFrame, dataFrame.columns, recordName, nameSpace, DefaultAvroSchemaConverter())
```

Comment:
Please keep the formatting consistnet with the rest of the project. For example this is correct indentation:
```Scala
def toAvroSchema(
  dataFrame: DataFrame,
  columnName: String,
  recordName: String = "topLevelRecord",
  nameSpace: String = "",
  schemaConverter: AvroSchemaConverter = DefaultAvroSchemaConverter()
): Schema = { ...
```